scraper.rb

```diff
@@ -4,12 +4,15 @@
 
 require 'pry'
 require 'require_all'
+require 'scraperwiki'
 
 require_rel 'lib'
 
-page = MembersPage.new('http://house.gov.by/ru/deputies-ru/')
+page = MembersPage.new(url: 'http://house.gov.by/ru/deputies-ru/')
+members = page.mp_page_urls.map do |url|
+  MemberPage.new(url: url)
+end
```

Comment:
it's not obvious here why you're building up a list of `members` first here, rather than just `.each`-ing over them as you go. 

If you really _do_ want to defer writing to the database until the very end, then it's probably better to do that in a single `save_sqlite` call (but doing it a loop is more in keeping with our normal approach).